### ISSUE TITLE
Searching for certain types of strings will result in an exception

### ISSUE THREAD

**Issue by xiaohutai** [bolt/bolt#7459]
```
An exception has been thrown during the rendering of a template ("Warning: preg_replace(): Unknown modifier '/'") in "search.twig" at line 51.
```
```
Warning: preg_replace(): Unknown modifier '/'
```

Details
-------

 - Relevant Bolt Version: [ 3.4 | 3.5 ] possibly all versions
 - Install type: [ Zip/tarball install | Composer install ]
 - PHP version: [ 5.6 | 7.0 ]
 - Used web server: [ Apache | Nginx ]

### Bug summary

Something with [`preg_replace`](http://php.net/manual/en/function.preg-replace.php) and escaped characters.

### Steps to reproduce

1. Install Bolt
2. [Optional] have some content
3. Search for `//`, e.g. navigate to `/search?search=//`.

### Expected result

No results found makes more sense than a Whoops (debug on) or Uncaught Exception (debug off).

### Actual result

![300-hours-in-photoshop](https://user-images.githubusercontent.com/4630335/39567592-c902fde8-4ebf-11e8-8951-97bf100baffb.png)

**Comment by GwendolenLynch** [comment]
Fixed in #7464

**Comment by xiaohutai** [comment]
@GawainLynch Thanks, mate 👍
